src/scripts/interaction.js

```diff
@@ -1598,6 +1610,16 @@ function Interaction(parameters, player, previousState) {
         });
 
         instance.on('click', function (event) {
+
+          if(instance.libraryInfo.machineName === "H5P.IVHotspot" && parameters.wrongAnswerParams.isWrongChoice) {
+            self.wrongAnswerClicked += 1;
```

Comment:
Rename to `wrongAnswerCount` to describe the purpose of the variable better.

Reply:
Fixed